### PR TITLE
Basic bdd improvement

### PR DESCRIPTION
**Pull request by yannickroger**
## Description
In order to test possible regressions of the param converter #129  , we needed to have the modified part covered by BDD.

### CODE REVIEW

**Comment by joaoinacio** [comment]
Looks OK, the failure is unrelated (some weird http cache issue, being looked into)

**Comment by yannickroger** [comment]
ping @bdunogier

**Comment by bdunogier** [comment]
Needs to be rebased on top of master to fix the HTTP cache issue.

**Comment by joaoinacio** [comment]
The "I am on {page} page" sentence not only checks/asserts, but also actually navigates to the page. (it is written as a "given" sentence):
```
$ bin/behat -p demo -d 'I am on' 
...
content | Given I am on/at (the) :page page
        | Visits the page identified by ':page', or the homepage.
        | Asserts that http response status code is not >= 400
        | at `EzSystems\DemoBundle\Features\Context\Content\Context::iAmOnPage()`
``` 

It is not a huge issue, but because this is already done previously, by url, there will be double requests for each feature.
IIRC this may be the case as well for the premium content feature.

**Comment by bdunogier** [comment]
+1 with @joaoinacio.

**Comment by bdunogier** [comment]
```
Given I am logged in as "subscriber" with password "publish"
 When I go to "/Getting-Started/Selected-Features/Getting-Started-with-eZ-Publish-Platform"

Given I am logged in as "member" with password "publish"
 When I go to "/Getting-Started/Selected-Features/Getting-Started-with-eZ-Publish-Platform"
```

**Comment by bdunogier** [comment]
Hmmm, I had misunderstood.

But... why ?

**Comment by yannickroger** [comment]
Updated with @joaoinacio 's remark. Thanks

Ready for merge @joaoinacio @bdunogier ?

File: Features/Content/place.feature [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Feature: Place
+
+    Scenario: As a visitor, I can see the place page
+        Given I am on the "place" page
+         And I see "eZ Mountains" text
```
Comment (joaoinacio) [inline Features/Content/place.feature]: Nitpick: `s/And/Then/`

File: Features/Content/feedback_form.feature [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Feature: Feedback form
+
+    Scenario: As a visitor, I can see the feedback form
+        Given I am on the "feedback_form" page
+         And I see "Contact Us" text
```
Comment (joaoinacio) [inline Features/Content/feedback_form.feature]: Nitpick: s/And/Then/

**Comment by yannickroger** [comment]
Thanks, fixed

**Comment by bdunogier** [comment]
You could really have changed the PR's title...

**Pull request merged**